### PR TITLE
Add EXPECT EQ

### PR DESCRIPTION
**Pull request by dj2**

### CODE REVIEW

File: src/amberscript/parser.cc [hunk 1]
```diff
@@ -995,6 +1019,41 @@ Result Parser::ParseClear() {
...
+                           const DatumType& type,
+                           std::vector<Value>* values) {
+  assert(values);
+
+  auto token = tokenizer_->NextToken();
+  while (!token->IsEOL() && !token->IsEOS()) {
+    Value v;
+
+    if ((type.IsFloat() || type.IsDouble())) {
+      if (!token->IsInteger() && !token->IsDouble()) {
```
Comment (dneto0) [inline src/amberscript/parser.cc]: IsInteger -> IsFloat

Comment (dj2) [reply src/amberscript/parser.cc]: The token doesn't have an IsFloat, just IsDouble. We ConvertToDouble a couple of lines down, so if we have an Integer or a Double then we're good to go. If we get anything other then a number then things go wrong.

Comment (dneto0) [reply src/amberscript/parser.cc]: ok.  I was wrong.

File: src/amberscript/parser.h [hunk 2]
```diff
@@ -64,6 +65,10 @@ class Parser : public amber::Parser {
   Result ParseClear();
   Result ParseExpect();
 
+  Result ParseValues(const std::string& name,
```
Comment (dneto0) [inline src/amberscript/parser.h]: Method comment please.  What is the name for?  Are we appending to *values or completely overwriting.

Comment (dj2) [reply src/amberscript/parser.h]: done

**Pull request merged**